include/k4a/k4a.hpp

```diff
@@ -904,8 +985,31 @@ class transformation
         }
     }
 
+    /** Transforms the depth image into 3 planar images representing X, Y and Z-coordinates of corresponding 3d points.
+     * Throws error on failure.
+     *
+     * \sa k4a_transformation_depth_image_to_point_cloud
+     * Creates a new image with the output.
+     */
+    image depth_image_to_point_cloud(const image &depth_image, k4a_calibration_type_t camera) const
+    {
+        image xyz_image = image::create(K4A_IMAGE_FORMAT_CUSTOM,
```

Comment:
The resolution of the xyz image needs to match the [resolution of the depth image](https://microsoft.github.io/Azure-Kinect-Sensor-SDK/develop/group___functions_ga7385eb4beb9d8892e8a88cf4feb3be70.html#ga7385eb4beb9d8892e8a88cf4feb3be70) passed in. 

One use case of this function is for the depth image to be transformed to the perspective of the color camera first, then converted to a point cloud. This allows you to align color values with the point cloud points.

Reply:
Oops, I got it all wrong. Thanks.